Cargo.toml

```diff
@@ -13,7 +13,7 @@ bit_field = "0.10"
 numeric-enum-macro = "0.2"
 buddy_system_allocator = "0.8"
 tock-registers = "0.8"
-aarch64-cpu = {git = "https://github.com/likey99/aarch64-cpu"}
+aarch64-cpu = "9.4.0"
```

Comment:
The newest `aarch64-cpu` has added what likey wants to add. So just use the official version.